src/core/client_channel/lb_metadata.cc

```diff
@@ -92,7 +91,7 @@ void MetadataMutationHandler::Apply(
       continue;
     }
     metadata->Append(key, std::move(value),
-                     [key](absl::string_view error, const Slice& value) {
+                     [key = key](absl::string_view error, const Slice& value) {
```

Comment:
What's the purpose of this change?

Reply:
The compiler complained without it.  Apparently, using a lambda capture on a structured binding is not legal until C++20.